config/mark-test-status.py

```diff
@@ -0,0 +1,10 @@
+from robot.libraries.BuiltIn import BuiltIn
+from robot.libraries.BuiltIn import _Misc
+import robot.api.logger as logger
+from robot.api.deco import keyword
+
+@keyword("TEST STATUS")
+def testStatus(status, reason):
+     driver = BuiltIn().get_library_instance('SeleniumLibrary').driver
+     driver.execute_script('browserstack_executor: {"action": "setSessionStatus", "arguments": {"status":"'+status+'", "reason": "'+reason+'"}}')
+     
```

Comment:
Please check if adding extra spaces/tabs at the last line, can remove that.

Reply:
Done under commit a2da1fdd2acf4b06f3fcf97b1840e0fea5ecb9c3